### ISSUE TITLE
Back up docs update: Add upgrade from mke3 section

### ISSUE THREAD

**Issue by sakshisharma84** [Mirantis/mke-docs#123]
The info on what to expect during upgrade from mke3 is missing from the back up docs. We should add al the relevant details.

**Comment by KoryKessel-Mirantis** [comment]
Please elaborate, @sakshisharma84. Glad to pick this up and integrate, but more information is needed. Are you the SME?

**Comment by sakshisharma84** [comment]
@KoryKessel-Mirantis No, I am not the SME. The existing backup docs don't talk about MKE3 vs MKE4 backup and what to expect during the upgrade. This information should be added so that the users are aware.

**Comment by KoryKessel-Mirantis** [comment]
@sakshisharma84, was this issue effectively handed by https://github.com/Mirantis/mke-docs/pull/260? If so, can we close it?

**Comment by sakshisharma84** [comment]
> [@sakshisharma84](https://github.com/sakshisharma84), was this issue effectively handed by [#260](https://github.com/Mirantis/mke-docs/pull/260)? If so, can we close it?

@KoryKessel-Mirantis sure! Thanks for the follow up.
